### PR TITLE
Update Linux Mint Ascii Logo

### PR DESCRIPTION
**Pull request by Gamegenorator**
## Description

Updated the Linux Mint ASCII Logo to Match Mint 19.3 and above.

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -7740,6 +7740,31 @@ EOF
...
+  :MMM${c1}:MMM:`                `:MMM:${c2}MMM:
+   .MMM${c1}.MMMM:--------------:MMMM.${c2}MMM.
+     '-MMMM${c1}.-MMMMMMMMMMMMMMM-.${c2}MMMM-'
+       '.-MMMM${c1}``--:::::--``${c2}MMMM-.'
+${c2}            '-MMMMMMMMMMMMM-'
+${c2}               ``-:::::-``
+EOF
+        ;;
+
+        "Linux Mint_old")
```
Comment (dylanaraps) [inline neofetch]:
```sh
"Linux Mint Old"* | "LinuxMintOld"* | "mint_old"*)
```

Comment (Gamegenorator) [reply neofetch]: Do I need to add `"Linux Mint Old"* | "LinuxMintOld"* | "mint_old"*)` to line 7765?

Comment (dylanaraps) [reply neofetch]: Replace the line with that, yeah.

**Comment by dylanaraps** [comment]
Thanks.

**Comment by Gamegenorator** [comment]
No Problem, Thanks for letting me be part of it all.

**Pull request merged**